woocommerce/woocommerce-ios
# [Blaze] Learn how to blaze sheet UI

Closes: #11566 


## Description
Presents the learn how sheet from the new Blaze intro screen.

## Testing instructions

- Create a JN site
- Create a new product and publish it
- From the Dashboard Blaze view tap "Create Campaign" button
- You should see the new intro view
- Tap "Learn how Blaze works" button
- You will see the new sheet

## Screenshots

| Light | Dark |
|--------|--------|
| ![Simulator Screen Recording - iPhone 14 Pro - 2023-12-29 at 18 44 54](https://github.com/woocommerce/woocommerce-ios/assets/524475/e5ba6183-beec-4212-b571-e1a52d655715) | ![Simulator Screen Recording - iPhone 14 Pro - 2023-12-29 at 18 45 09](https://github.com/woocommerce/woocommerce-ios/assets/524475/d85ea084-9774-454e-ab8f-2f36a0ceddd1) | 

---
